src/NServiceBus.Core/Serialization/SerializationSettingsExtensions.cs

```diff
@@ -0,0 +1,48 @@
+namespace NServiceBus
+{
+    using System;
+    using System.Collections.Generic;
+    using Serialization;
+    using Settings;
+
+    static class SerializationSettingsExtensions
+    {
+        const string AdditionalSerializersSettingsKey = "AdditionalDeserializers";
```

Comment:
I disagree, the settings are available from everywhere. The setting is provided by the feature and the extensions only add convenience to it. I think we mostly define they keys in the feature.